### ISSUE TITLE
Doesn't work on Ventura 13.2

### ISSUE THREAD

**Issue by jarm9** [fikovnik/ShiftIt#332]
<img width="555" alt="Captura de pantalla 2023-01-25 a la(s) 15 58 55" src="https://user-images.githubusercontent.com/93294692/214701264-b9559771-da3e-45c2-83e7-809f32ca5513.png">
The option is enabled but when giving recheck it shows the message that it is not enabled yet

**Comment by SodY2** [comment]
can confirm this behaviour on mac os 13.2

**Comment by jarm9** [comment]
Yes, this issue is presented on Ventura 13.2 (22D49) :(

**Comment by jarm9** [comment]
> can confirm this behaviour on mac os 13.2

Yes, this issue is presented on Ventura 13.2 (22D49) :(

**Comment by DevotionGeo** [comment]
Works fine for me on Ventura 13.2 (M1).

**Comment by eduardoroliveira** [comment]
Try restarting your Mac. It solved my problem!

**Comment by sleeke** [comment]
Restarting also solved for me (Ventura 13.2.1 (22D68), M1)

**Comment by rewolf** [comment]
Yeah for me too it's been working for the most part. Occasionally I had noticed it not working, but didn't correlate it with anything. 
But in a Zoom meeting first thing this morning, I was unable to use ShiftIt (my computer had restarted itself over night and windows needed rearranging).  But since ending the meeting I have control again.
